### PR TITLE
storage: Mock and ignore usage for pixel tests

### LINKED ISSUE #19679: Nightly tests did not succeed on fedora-39/updates-testing: TestStorageBasic.testBasic flaky pixels due to column widths
Tests failed on f0b096151519c1764335f25569a9c4120b0facf3, [logs](https://cockpit-logs.us-east-1.linodeobjects.com/pull-0-20231202-012546-f0b09615-fedora-39-updates-testing/log.html)
[bodhi updates](https://bodhi.fedoraproject.org/updates/?search=&status=testing&status=stable&releases=F39)

### PR DESCRIPTION
**Pull request by mvollmer**
Fixes #19679

### CODE REVIEW

File: test/verify/check-storage-basic [hunk 1]
```diff
@@ -69,8 +69,12 @@ class TestStorageBasic(storagelib.StorageCase):
         b.wait_visible(self.card("Storage"))
         b.wait_visible(self.card_row("Storage", name=disk))
         b.assert_pixels(self.card("Storage"), "overview",
-                        # Usage numbers for the root filesystem are not stable
-                        ignore=["[data-test-row-location='/']"])
+                        # Usage numbers are not stable and also cause
+                        # the table columns to shift. The usage bars
+                        # are not stable but are always the same size,
+                        # so it is good enough to ignore them.
+                        mock={".usage-text": "---"},
```
Comment (martinpitt) [inline test/verify/check-storage-basic:76]: I'd have prefered a realistic mock value which represents the right size, like "1.2 / 34 GB" or so. That should better point out wrapping problems with narrow layouts.

But details, good enough!

**Review by martinpitt — APPROVED** [review]
Danke!

**Comment by mvollmer** [comment]
Another idea is to do the core pixel test stuff in a single JavaScript function: mocking, clip and ignore rectangle computation, screen capture. Then it should all be atomic. Layout needs to run after mocking, but I think that will happen.

File: pkg/storaged/pages.jsx [hunk 2]
```diff
@@ -501,7 +501,7 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
 
         if (typeof size === "number") {
             if (narrow)
-                size = fmt_size(size);
+                size = <span className="usage-text">{fmt_size(size)}</span>;
```
Comment (cockpituous) [inline pkg/storaged/pages.jsx:504]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19685-20231204-165545-0123808a-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
That looks much better! The remaining failures are unrelated and well-known flakes, retrying.

**Review by martinpitt — APPROVED** [review]

**Pull request merged**